automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala

```diff
@@ -164,8 +164,8 @@ class MethodLaunchSpec extends FreeSpec with ParallelTestExecution with Matchers
               eventually {
                 val status = submissionDetailsPage.getApiSubmissionStatus(billingProject, workspaceName, submissionId)
                 logger.info(s"Status is $status in Submission $billingProject/$workspaceName/$submissionId")
-                withClue(s"Monitoring Submission $billingProject/$workspaceName/$submissionId. Status is Done or Running.") {
-                  status should (equal("Done") or equal ("Running"))
+                withClue(s"Monitoring Submission $billingProject/$workspaceName/$submissionId. Status is Done or Submitted.") {
```

Comment:
Could we have a comment here along the lines of:
> // Submitted means that Cromwell has been passed the workflows. Submission statuses do not include a concept of `Running`

to prevent this confusion resurfacing next time this test comes in for reappraisal?

Reply:
Submitted means that Cromwell has been passed the workflows Is not a true statement